### PR TITLE
KAN-13 and KAN-19: made CreatePostPage and added Karla font

### PR DESCRIPTION
**Pull request by shermanto24**

### CODE REVIEW

File: adsh/src/App.js [hunk 1]
```diff
@@ -1,10 +1,12 @@
 import './styles/App.css';
 import Home from './pages/Home';
+import CreatePostPage from './pages/CreatePostPage'
 
 function App() {
   return (
   <Home/>
   )
+  //return CreatePostPage()
```
Comment (DFLotus) [inline adsh/src/App.js]: Remove comment

File: adsh/src/pages/CreatePostPage.js [hunk 2]
```diff
@@ -0,0 +1,37 @@
+import React from "react";
+import '../styles/CreatePostPage.css'
+
+export default function CreatePostPage() {
```
Comment (DFLotus) [inline adsh/src/pages/CreatePostPage.js]: Please follow same coding style for functions

File: adsh/src/pages/CreatePostPage.js [hunk 3]
```diff
@@ -0,0 +1,37 @@
...
+    return (
+        <div className='create-post-page'>
```
Comment (DFLotus) [inline adsh/src/pages/CreatePostPage.js]: Please add Nav bar

Comment (hninl23) [reply adsh/src/pages/CreatePostPage.js]: I can add the NavBar later b/c I did something similar to it for the login page

**Review by DFLotus — APPROVED** [review]
LGMT!

**Pull request merged**